MailCore/Models/Thread.swift

```diff
@@ -45,6 +45,7 @@ public class Thread: Object, Decodable, Identifiable {
     @Persisted public var flagged: Bool
     @Persisted public var answered: Bool
     @Persisted public var forwarded: Bool
+    @Persisted public var folderId = ""
```

Comment:
We could easily create a migration to populate this field, would we benefit from it ?